### PR TITLE
Add TIFF support to matplotlib.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/66/
Test (probably) passed

File: packages/package_tiff_4_0_3/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="tiff" version="4.0.3">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">ftp://ftp.remotesensing.org/pub/libtiff/tiff-4.0.3.tar.gz</action>
+                <action type="shell_command">./configure --prefix=$INSTALL_DIR</action>
+                <action type="make_install" />
+                <action type="set_environment">
```
Comment (bgruening) [inline packages/package_tiff_4_0_3/tool_dependencies.xml]: `<action type="autoconf" />` should replace make_install and shell_command

**Comment by bgruening** [comment]
Thanks @davebx !

**Comment by erasche** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://gx.hx42.org//job/IUC-Github/81/
Test (probably) passed

**Pull request merged**